pages/index.tsx

```diff
@@ -15,7 +15,7 @@ export default function HomePage() {
   const userId = useUserId()
 
   return (
-    <div className="max-sm:flex-col gap-8 lg:gap-12 flex justify-center px-4 lg:pt-4 mx-auto max-w-6xl">
+    <div className="max-md:flex-col gap-8 lg:gap-12 flex justify-center px-4 lg:pt-4 mx-auto max-w-6xl">
```

Comment:
Looks like at >sm <md size, the track record is shown below the questions. I think it should be hidden instead (the bottom nav is shown so you can use the stats page)
<img width="885" alt="image" src="https://github.com/user-attachments/assets/d10dabab-2ad2-4d6c-900e-242e2563c82f">